libraries/AP_NavEKF3/AP_NavEKF3_PosVelFusion.cpp

```diff
@@ -240,6 +246,28 @@ bool NavEKF3_core::resetHeightDatum(void)
     return true;
 }
 
+/*
+  correct GPS data for position offset of antenna phase centre relative to the IMU
+ */
+void NavEKF3_core::CorrectGPSForAntennaOffset(gps_elements &gps_data)
```

Comment:
[Redundant Code] Hi. Can this method be moved to NavEKF_core_common.h/.cpp, the common parent class? Sorry if I misunderstand, the same piece of code has been used both in  NavEKF3_core and NavEKF2_core.

Reply:
gps_elements is defined as a local structure in each class, so we'd have to move that up as well. We could do that, but Paul may prefer to keep things more separated

Reply:
In that case, lets go ahead. :D